extensions/blocks/pinterest/pinterest.php

```diff
@@ -180,10 +218,28 @@ function render_amp_pin( $attr ) {
  * @return string
  */
 function load_assets( $attr, $content ) {
+	if ( ! jetpack_is_frontend() ) {
+		return $content;
+	}
 	if ( Blocks::is_amp_request() ) {
 		return render_amp_pin( $attr );
 	} else {
+		$url  = $attr['url'];
+		$type = pin_type( $url );
+
+		if ( ! $type ) {
+			return null;
+		}
+
 		wp_enqueue_script( 'pinterest-pinit', 'https://assets.pinterest.com/js/pinit.js', array(), JETPACK__VERSION, true );
-		return $content;
+		return sprintf(
+			'
+			<div class="wp-block-jetpack-pinterest">
+				<a data-pin-do="%1$s" href="%2$s"></a>
+			</div>
+		',
+			$type,
+			$url
```

Comment:
We should also escape this:
```suggestion
			esc_url( $url )
```